src/huntsman/pocs/scheduler/field.py

```diff
@@ -0,0 +1,142 @@
+from astroplan import FixedTarget
+from astropy.coordinates import SkyCoord
+
+from panoptes.utils.library import load_module
+from panoptes.pocs.base import PanBase
+from huntsman.pocs.utils import dither
+
+
+class AbstractField(PanBase):
```

Comment:
Do we wanna add the ability to change the exptime for each sub-field? Might be useful for dithering on/off a PSF reference bright star to get optimal PSF wing subtraction.

Reply:
Would be nice but that doesn't work with the current `scheduler`. Definitely for future work though, as also useful for flat fields.